MANIFEST.in

```diff
@@ -1,8 +1,7 @@
 recursive-include doc *
 prune doc/_build
 recursive-include examples *
-include README LICENSE INSTALL
-include setup.py pexpect.py fdpexpect.py pxssh.py screen.py ANSI.py FSM.py
-include test.env tools/testall.py
+include README.rst LICENSE
+include setup.py pexpect/bashrc.sh
```

Comment:
I'm pretty sure setup.py doesn't need to be listed here - it will include itself.